Traceback (most recent call last):
  File "/ysm-gpfs/pi/gerstein/gamze/CZI/encode/code/genome/getSeq_wN.py", line 52, in <module>
    fileB = open(sys.argv[3], "r")
IndexError: list index out of range
[E::sam_parse1] missing SAM header
[W::sam_read1] parse error at line 1
[main_samview] truncated file.
Traceback (most recent call last):
  File "/ysm-gpfs/pi/gerstein/gamze/CZI/encode//code/genome/getSeq_woN.py", line 35, in <module>
    print("%s" % header[i])
IOError: [Errno 32] Broken pipe
[W::bam_merge_core2] No @HD tag found.
